dashboard/app/views/pd/application/teacher_application_mailer/principal_approval_completed_teacher_receipt.html.haml

```diff
@@ -0,0 +1,19 @@
+%p
+  Dear
+  = @application.first_name + ','
+
+%p
+  Your Administrator/School Leader has approved your application. Your application
+  is now considered complete and we will review it and get back to you in 2-4 weeks.
+
```

Comment:
Nit: I know in other emails and parts of the application we've said "2 to 4" instead of "2-4". Super minor change so it would only be for consistency but not a big deal if you want to just merge it!